### ISSUE TITLE
Can not add files to "/storage/emulated/0" directory

### ISSUE THREAD

**Issue by jankoJelic** [itinance/react-native-fs#1082]
Hello I have been using:
"react-native": "0.65.1"
"react-native-fs": "^2.18.0"

I can RNFS.mkdir to "storage/emulated/0/Download/myApp", but I can not .mkdir "storage/emulated/0/myApp"

 [Error: Directory could not be created]
 
 I have both write and read permissions, and it works perfectly in subfolders, but I want to create in in the main directory like some other apps already do on my device.

**Comment by anwersolangi** [comment]
Did you find any solution??? I am also struggling with this too

**Comment by jankoJelic** [comment]
No, still having the same problem. I have decided to put everything into 0/Android folder

**Comment by anwersolangi** [comment]
> No, still having the same problem. I have decided to put everything into 0/Android folder

That's good, but not user-friendly as other apps, also I'm thinking of moving to another library

**Comment by jankoJelic** [comment]
I already tried rn-fetch-blob but the same issue occurs. So something else could be the problem.

**Comment by utiliter** [comment]
I have the same problem. please fix this!

**Comment by NasirAbbasiWebHR** [comment]
I have same issue.

**Comment by ianic2** [comment]
I also have the same issue.

**Comment by RupmalyaK** [comment]
Same here

**Comment by desenvolvedorHK** [comment]
Hey guys, does anyone found any solution? Im with that problem in some devices with Android 11

**Comment by fkhrrz** [comment]
i'm having same problem for android 11+

**Comment by Leslie717** [comment]
please help us!

**Comment by hms-douglas** [comment]
same problem here

**Comment by VitaminC1989** [comment]
I'm same problem too.
my expect is that download file that from url to `/storage/emulated/0/Downloads`  but file manager of mobile system can't  scan it. I have to  use Professional file manager to find the file that downloaded into  path  `/storage/emulated/0/Downloads`.
  
This Q&A provides a method to solve such problems that **write file to Android external storage**
https://stackoverflow.com/questions/63392170/write-files-to-the-android-external-storage-using-react-native-fs

![Uploading pasteboard-image_2023-12-11 06-51-20.png…]()
